### PR TITLE
okcoin exchange

### PR DESCRIPTION
**Pull request by xhad**
Okcoin is working well here, for me. Still need to add child-accounts. Please take a look

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,6 +29,7 @@
     "poloniex.js": "0.0.6",
     "semver": "2.2.1",
     "sqlite3": "3.1.3",
+    "websocket": "latest",
```
Comment (askmike) [inline package.json:32]: unfortunately this does not work on windows. I'll try to find an alternative which is more cross platform.

**Comment by askmike** [comment]
thanks a lot! Will give this some proper testing.

**Comment by LobodaCrypto** [comment]
Hello, askmike
Could you add usd/btc, usd/eth, btc/eth, btc/neo, trading pair in Okcoin exchange?

**Comment by LobodaCrypto** [comment]
can I use gekko ui for okcoin exchange?

**Pull request merged**